test-apps/display-performance-test-app/src/frontend/TestConfig.ts

```diff
@@ -115,10 +115,10 @@ export interface TestConfigProps {
    * Default: "*"
    */
   iModelName?: string;
-  /** The name of the iModelHub project from which to obtain iModels. Currently not supported.
+  /** The name of the iTwin from which to obtain iModels. Currently not supported.
    * Default: "iModel Testing"
    */
-  iModelHubProject?: string;
+  iModelHubITwin?: string;
```

Comment:
No. This is a JSON format, you cannot change the property names or you will break all existing JSON files.

Reply:
Reverting the change. What is the plan for updating this property and the JSON files in the future?

Reply:
There is no plan to do so. This is an internal testing tool, we care far more about not breaking our existing config files than we do about "iTwin" vs "project".